### ISSUE TITLE
There is no way to add custom thumb composable in slider

### ISSUE THREAD

**Issue by ajitsinghrathore** [2307vivek/Seeker#51]
In default compose slider , we can change thumb composable to any composable depending on the usecase , but I think we don't have the option here . Is there any plan to add that in future ?

**Comment by 2307vivek** [comment]
Yes! There are plans to add a feature for custom thumb among other things.
